PowerPointLabs/PowerPointLabs/PictureSlidesLab/Service/StylesDesigner.cs

```diff
@@ -41,7 +41,7 @@ public sealed class StylesDesigner : PowerPointPresentation, IStylesDesigner
         public StylesDesigner(Application app = null)
         {
             Path = TempPath.TempFolder;
-            Name = "PictureSlidesLabPreview" + Guid.NewGuid().ToString().Substring(0, 7);
```

Comment:
I intentionally made it w/o extension, in case users see it as a ppt file and open it. any error u encountered when w/o extension?